bootstrap/api/v1alpha1/rke2config_types.go

```diff
@@ -328,6 +333,18 @@ type SecretFileSource struct {
 	Key string `json:"key"`
 }
 
+// ConfigMapFileSource adapts a ConfigMap into a FileSource.
```

Comment:
We don't add new features to previous API versions, this can be removed from v1alpha1.